contentcuration/contentcuration/static/js/bundle_modules/base.js

```diff
@@ -1,13 +1,45 @@
 var attachfastclick = require("fastclick");
 var $ = require("jquery");
+var get_cookie = require("utils/get_cookie");
 global.$ = $;
 global.jQuery = $;
+require("handlebars/helpers");
 
-require("bootstrap/less/bootstrap.less");
-require("bootstrap/dist/js/npm.js");
-require("bootstrap/dist/css/bootstrap.css");
 require("../../less/styles.less");
-require("../../less/global-variables.less");
+require("bootstrap/dist/js/npm.js");
+
+var csrftoken = get_cookie("csrftoken") || "";
+
+function csrfSafeMethod(method) {
+    // these HTTP methods do not require CSRF protection
+    return (/^(GET|HEAD|OPTIONS|TRACE)$/.test(method));
+}
+
+function sameOrigin(url) {
+    // test that a given url is a same-origin URL
+    // url could be relative or scheme relative or absolute
+    var host = document.location.host; // host + port
+    var protocol = document.location.protocol;
+    var sr_origin = '//' + host;
+    var origin = protocol + sr_origin;
+    // Allow absolute or scheme relative URLs to same origin
+    return (url == origin || url.slice(0, origin.length + 1) == origin + '/') ||
```

Comment:
For string string comparisons it is fine, but it might be better for consistency. I forget whether jshint treats this harshly or not.